pkg/render/manager.go

```diff
@@ -94,15 +95,20 @@ func Manager(
 	clusterDomain string,
 	esLicenseType ElasticsearchLicenseType,
 ) (Component, error) {
-	tlsSecrets := []*corev1.Secret{tlsKeyPair}
-	tlsSecrets = append(tlsSecrets, secret.CopyToNamespace(ManagerNamespace, tlsKeyPair)...)
+	var tlsSecrets []*corev1.Secret
 	tlsAnnotations := make(map[string]string)
+	if installation.CertificateManagement == nil {
```

Comment:
```suggestion
	if installation != nil && installation.CertificateManagement == nil {
```

Reply:
It is impossible for installation to be nil. This is true for everywhere installation is used in the render package.